packages/frontend/src/components/Form/ReportForm/ReportForm.tsx

```diff
@@ -87,15 +87,31 @@ const ReportForm: React.FC<ReportFormProps> = ({ closeModal, notifyParentAboutSu
 
     // Calculate possible stations based on entity, line, station, and search input
     const possibleStations = useMemo(() => {
-        let stations = allStations
+        // Sort station records from a station record [stationKey, station] array by station name
+        const sortStationRecordsByStationName = (
+            recordA: (string | StationProperty)[],
+            recordB: (string | StationProperty)[]
+        ) => {
+            const stationA = recordA[1] as StationProperty
+            const stationB = recordB[1] as StationProperty
+            if (stationA.name > stationB.name) return 1
+            if (stationA.name < stationB.name) return -1
+            return 0
+        }
+
+        const sortedAllStations = Object.fromEntries(Object.entries(allStations).sort(sortStationRecordsByStationName))
+        let stations = sortedAllStations
+
         if (currentStation) {
             stations = { [currentStation]: allStations[currentStation] }
         } else if (currentLine) {
             stations = Object.fromEntries(
-                allLines[currentLine].map((stationKey) => [stationKey, allStations[stationKey]])
+                allLines[currentLine]
+                    .map((stationKey) => [stationKey, allStations[stationKey]])
+                    .sort(sortStationRecordsByStationName)
```

Comment:
Once a line is selected, the stations naturally follow the train's traversal order along the line. I'm not entirely sure if people are faster if this order is disrupted, but I’d like to test this hypothesis. In my opinion, the best approach is to remove this line, merge and deploy your changes, and observe whether people can report inspectors more quickly. Once we’ve established a new baseline, we can reintroduce the line to test the hypothesis.

Reply:
Oh that's such a great insight! I would have never known this. Personally, I also prefer keeping the station order as it is in real life. It makes more sense as I assume most riders using this app are regulars, and are familiar with that order. But please keep me posted if you ever decide to test it out.

Reply:
will do